src/types/mapReduce.assert.ts

```diff
@@ -0,0 +1,33 @@
+import { ObjectId } from 'mongodb'
+import * as ta from 'type-assertions'
+import { TsMapReduceOptions } from './mapReduce'
```

Comment:
You should test `.mapReduce`

Reply:
Sure, just wanted to confirm if you are referring to writing Type Assertions or Jest tests.